docs/config_search.md

```diff
@@ -147,57 +210,77 @@ profile_models:
+
+---
+
+_An example model analyzer YAML config that performs a Quick Search:_
+
+```yaml
+model_repository: /path/to/model/repository/
+
+run_config_search_mode: quick
+profile_models:
+  - model_A
+```
+
+---
+
+### **Limiting Batch Size, Instance Count, and Client Concurrency**
+
+Using the `--run-config-search-<min/max>...` config options you have the ability to clamp the algorithm's upper or lower bounds for the model's batch size and instance count, as well as the client's request concurrency.
+
+_Note: By default, quick search runs unbounded and ignores any default values for these settings_
+
+---
 
-### Limiting Batch Size, Instance Count, and Client Concurrency
+_An example model analyzer YAML config that performs a Quick Search and constrains the search parameters:_
 
-Using the `--run-config-search-<min/max>...` CLI options you have the ability to clamp the algorithm's upper or lower bounds for the model's batch size and instance count, as well as the client's concurrency.
+```yaml
+model_repository: /path/to/model/repository/
+
+run_config_search_mode: quick
+run_config_search_max_instance_count: 4
+run_config_search_max_concurrency: 16
+run_config_search_min_model_batch_size: 8
+
+profile_models:
+  - model_A
+```
 
-Note: That by default quick search runs unbounded and ignores any default values for these settings
+---
 
 ## Ensemble Model Search
 
 _This mode has the following limitations:_
 
 - Can only be run in `quick` search mode
-- Only supports up to 3 sub-models
+- Only supports up to 4 sub-models
 
-Non-BLS Ensemble models can be optimized using the Quick Search mode's hill climbing algorithm to search the ensemble sub-model's configuration spaces in parallel, looking for the maximal objective value within the specified constraints. Model Analyzer has observed positive outcomes towards finding the maximum objective value; with runtimes under one hour (compared to the days it would take a brute force run to complete) for ensembles with up to three submodels.
+Ensemble models can be optimized using the Quick Search mode's hill climbing algorithm to search the ensemble sub-model's configuration spaces in parallel, looking for the maximal objective value within the specified constraints. Model Analyzer has observed positive outcomes towards finding the maximum objective value; with runtimes under one hour (compared to the days it would take a brute force run to complete) for ensembles with up to four submodels.
 
-After it has found the best config(s), it will then sweep the top-N configurations found (specified by `--num-configs-per-model`) over the concurrency range before generation of the summary reports.
+After MA has found the best config(s), it will then sweep the top-N configurations found (specified by `--num-configs-per-model`) over the concurrency range before generation of the summary reports.
 
 ## Multi-Model Search Mode
```

Comment:
The other sections have a thick line before them. Should one be here as well?

Reply:
Yes. Fixed